package.json

```diff
@@ -12,7 +12,8 @@
     "redux": "^3.6.0",
     "redux-devtools-extension": "^1.0.0",
     "redux-logger": "^2.7.4",
-    "redux-thunk": "^2.1.0"
+    "redux-thunk": "^2.1.0",
+    "zoo-grommet": "^0.1.0"
   },
```

Comment:
By the way, the Pivotal UI modules are also installed as dev dependencies, not production. Should that be changed too?